spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/integration/outbound/PubSubMessageHandler.java

```diff
@@ -62,10 +63,10 @@
 
 	private HeaderMapper<Map<String, String>> headerMapper = new PubSubHeaderMapper();
 
-	public PubSubMessageHandler(PubSubOperations pubSubTemplate, String topic) {
-		Assert.notNull(pubSubTemplate, "Pub/Sub template cannot be null.");
+	public PubSubMessageHandler(PubSubOperations pubSubPublisherOperations, String topic) {
```

Comment:
Yeah... You didn't get me.
This one must be a `PubSubPublisherOperations` as well 😉

Reply:
missed it